forms-flow-bpm/src/main/java/org/camunda/bpm/extension/hooks/listeners/SendSubmissionToODSDelegate.java

```diff
@@ -129,4 +136,39 @@ public Map<String, Object> convertToSnake(Map<String, Object> values) {
         }
         return map;
     }
+
+    public Map<String, Object> replaceTextAll(Map<String, Object> values, String[][] replaceArray) {
+        Map<String, Object> map = new HashMap<>();
+        for (var entry : values.entrySet()) {
+            if (entry.getValue() instanceof String) {
+                String temp = entry.getValue().toString();
+                for (int i = 0; i < replaceArray.length; i++) {
+                    if (replaceArray[i].length > 1) {
+                        temp = temp.replaceAll(
+                            Pattern.quote(replaceArray[i][0]), 
+                            Matcher.quoteReplacement(replaceArray[i][1]));
+                    }
+                }
+                map.put(entry.getKey(), temp);
+            } else {
+                map.put(entry.getKey(), entry.getValue());
+            }
+        }
+        return map;
+    }
+
+    public static String[][] parseStringToArray(String[] input) {
+        // It covert string array from[','',@,#] to [[',''],[@,#]].
+        int replaceArraylength = input.length/2;
+        String[][] nestedArray = new String[replaceArraylength][2];
+        int init = 0;
+        for (int i = 0; i < input.length; i += 2) {
+            if (i+1 <= input.length && init < replaceArraylength) {
+                nestedArray[init][0] = input[i];
+                nestedArray[init][1] = input[i+1];
+                init++;
+            }
+        }
+        return nestedArray;
+    }
 }
```

Comment:
I'm wondering if we change the input type in the workflow from string to an array of strings like ["@, #", "-,+"] makes things easier for the form designer. However, I'm unsure how hard it is to escape ' and " in that case.

Reply:
@iman-jamali-fw I have moved from string array to list. apply your suggestion it and pushed new code.

Reply:
Looks awsome!